hudi-cli/src/test/java/org/apache/hudi/cli/common/HoodieTestCommitOperate.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.cli.common;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.hudi.avro.model.HoodieWriteStat;
+import org.apache.hudi.common.model.HoodieCommitMetadata;
+import org.apache.hudi.common.model.HoodieRollingStatMetadata;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Utility methods to commit instant for test.
+ */
+public class HoodieTestCommitOperate {
+
+  /**
+   * Converter HoodieCommitMetadata to avro format and ordered by partition.
+   */
+  public static org.apache.hudi.avro.model.HoodieCommitMetadata commitMetadataConverterOrdered(
+      HoodieCommitMetadata hoodieCommitMetadata) {
+    return orderCommitMetadata(commitMetadataConverter(hoodieCommitMetadata));
+  }
+
+  /**
+   * Converter HoodieCommitMetadata to avro format.
+   */
+  public static org.apache.hudi.avro.model.HoodieCommitMetadata commitMetadataConverter(
```

Comment:
> IIUC, `converter` is a noun while `convert` is a verb. A method usually means a behavior that starts with a verb actively. So considering the function of this method, wdyt about renaming to `convertCommitMetadata`? It's an open topic, you can share your thought if you have.

Actually, it is a copy of HoodieCommitArchiveLog$commitMetadataConverter[https://github.com/hddong/incubator-hudi/blob/Add-test-CommitsCommand/hudi-client/src/main/java/org/apache/hudi/table/HoodieCommitArchiveLog.java#L334]. Since HoodieCommitArchiveLog construct with `HoodieTableMetaClient` and `HoodieWriteConfig`, but test class and this mothod not need.

Reply:
IMO, we can move [HoodieCommitArchiveLog$commitMetadataConverter] to `ClientUtils`, wdyt.

Reply:
IMO, we can just make `HoodieCommitArchiveLog#commitMetadataConverter` a **static** method and rename it to `convertCommitMetadata` then you reuse it. WDYT?

Reply:
> IMO, we can just make `HoodieCommitArchiveLog#commitMetadataConverter` a **static** method and rename it to `convertCommitMetadata` then you reuse it. WDYT?

Yes, this is  a better solution.